compute/cker/include/cker/operation/DepthwiseConv.h

```diff
@@ -180,6 +182,47 @@ inline void DepthwiseConv(const DepthwiseConvParams &params, const Shape &input_
   cpu_backend_threadpool::Execute(tasks.size(), tasks.data(), ruy_context);
 }
 
+template <typename T>
+void DepthwiseConvOp(const DepthwiseConvParams &params, const Shape &input_shape,
+                     const T *input_data, const Shape &filter_shape, const T *filter_data,
+                     const Shape &bias_shape, const T *bias_data, T *padded_filter_data,
+                     bool pad_filter, T *filter_buffers_data, const Shape &output_shape,
+                     T *output_data)
+{
+  if (params.stride_height != params.stride_width)
+    throw std::runtime_error("Not support different length strides");
+
+  if (params.dilation_height_factor != 1 || params.dilation_width_factor != 1)
+    throw std::runtime_error{"Not support dilation other than 1."};
+
+  const int batch = MatchingDim(input_shape, 0, output_shape, 0);
+  const int input_depth = input_shape.Dims(3);
+  const int output_depth = output_shape.Dims(3);
+  const int input_height = input_shape.Dims(1);
+  const int input_width = input_shape.Dims(2);
+  const int filter_height = filter_shape.Dims(1);
+  const int filter_width = filter_shape.Dims(2);
+  const int output_height = output_shape.Dims(1);
+  const int output_width = output_shape.Dims(2);
+  const int stride = params.stride_height;
+  const int depth_multiplier = params.depth_multiplier;
+  const int pad_height = params.padding_values.height;
+  const int pad_width = params.padding_values.width;
+  const T activation_min = params.float_activation_min;
+  const T activation_max = params.float_activation_max;
+
+  depthwise_conv_op::LaunchDepthwiseConvOp<Eigen::ThreadPoolDevice, T>()(
+    batch, input_height, input_width, input_depth, filter_height, filter_width, depth_multiplier,
+    stride, pad_height, pad_width, output_height, output_width, output_depth, input_data,
+    filter_data, padded_filter_data, pad_filter, filter_buffers_data, output_data);
+
+  if (bias_data != nullptr)
```

Comment:
i think it would be better to add a exception action for the case where bias_data is nullptr.
for example, add assert (bias_data) as follows refer to biasHelper,
```
template <typename T>
void biasHelper(const Shape &bias_shape, const T *bias_data, const Shape &input_shape,
                T *input_data, T activation_min, T activation_max)
{
  int channel_dim = input_shape.DimensionsCount() - 1;

  assert(input_shape.Dims(channel_dim) == bias_shape.Dims(0));
  assert(input_data);
  assert(bias_data);
```

Reply:
Does the circle model always have `bias_data` in DepthwiseConv2D operation? The TensorFlow API supports the `use_bias` attribute for null bias data. I am not sure that `bias_data` is not always nullptr.

Reply:
@jyoungyun You are right. Tensorflow API supports optional bias. But onert does not support optional bias for DepthwiseConv2D op yet. So, if a circle model has it, that model won't run on onert.
Anyway, I think it's better to a test that does not have bias data if you'd like to support the optional bias on cker.